src/utils/VerificationValidationParser.cpp

```diff
@@ -155,6 +155,98 @@ void Parser::searchFinalDefense(Result* r) {
+    Test* type = nullptr;
+
+    if (QString::compare(DefaultTests::NO_NULL_REGIONS.testCommand, cmd, Qt::CaseInsensitive) == 0)
+        type = (Test*) &(DefaultTests::NO_NULL_REGIONS);
+    
+    else if (QString::compare(DefaultTests::NO_OVERLAPS.testCommand, cmd, Qt::CaseInsensitive) == 0)
+        type = (Test*) &(DefaultTests::NO_OVERLAPS);
+
+    // search for DB errors (if found, return)
+    if (Parser::gqaDBNotFoundErrors(r)) return r;
+    
+    QStringList lines = r->terminalOutput.split('\n');
+    bool startParsing = false;
+    for (size_t i = 0; i < lines.size(); i++) {
+        // if no usage errors, run specific test
+        if(lines[i].startsWith("list Overlaps"))
+        {
+            startParsing = true;
+            continue;
+        }
+        if (!Parser::gqaCatchUsageErrors(r, lines[i]) && type && startParsing)
+            Parser::gqaSpecificTest(r, lines[i], type);
+    }
+
+    // final defense: find any errors / warnings
+    if (r->resultCode == Result::Code::PASSED)
+        Parser::gqaFinalDefense(r);
+
+    return r;
+}
+
+void Parser::gqaSpecificTest(Result* r, const QString& currentLine, const Test* type) {
+    if (currentLine.trimmed().isEmpty()) return;
+    
+    if(type == &DefaultTests::NO_NULL_REGIONS)
+    {
+        QString objectPath1 = currentLine.split(' ')[0];
+        QString objectName1 = objectPath1;
+        int slashIdx1 = objectPath1.lastIndexOf('/');
+        if(slashIdx1 != -1)
+            objectName1 = objectPath1.mid(slashIdx1 + 1, objectPath1.size() - slashIdx1 - 1);
+
+        if(currentLine.contains("was not hit"))
+        {
+            r->resultCode = Result::Code::FAILED;
+            r->issues.push_back({objectName1, "Was not hit"});
+        }
+    }
+
+    else if(type == &DefaultTests::NO_OVERLAPS)
+    {
+        if(currentLine.contains("was not hit")) return;
+
+        r->resultCode = Result::Code::WARNING;
+        QStringList splitLine = currentLine.split(' ');
+
+        QString objectPath1 = splitLine[0];
+        QString objectName1 = objectPath1;
+        int slashIdx1 = objectPath1.lastIndexOf('/');
+        if(slashIdx1 != -1)
+            objectName1 = objectPath1.mid(slashIdx1 + 1, objectPath1.size() - slashIdx1 - 1);
+
+        QString objectPath2 = splitLine[1];
+        QString objectName2 = objectPath2;
+        int slashIdx2 = objectPath2.lastIndexOf('/');
+        if(slashIdx2 != -1)
+            objectName2 = objectPath2.mid(slashIdx2 + 1, objectPath2.size() - slashIdx2 - 1);
+
+        QString countString = splitLine[2];
+        QString distanceString = splitLine[3];
+        int parenthIdx = currentLine.indexOf('(');
+        QString locationString = currentLine.mid(parenthIdx + 1, currentLine.size() - parenthIdx - 1);
+
+        r->issues.push_back({objectName1, "Overlaps with '" + objectName2 + "' " + countString});
+    }
+    
+}
+
+
+bool Parser::gqaCatchUsageErrors(Result* r, const QString& currentLine) {
+    return false;
```

Comment:
this should be implemented in case user creates custom command and messes up syntax